docs/tools/palette_conv.md

```diff
@@ -0,0 +1,120 @@
+# Palette Conversion Tool
+
+The palette_conv tool allows you to convert between various palette formats for use with the ACE engine. This document explains how to use the palette_conv tool and how to integrate palette conversion into your build process.
+
+## Supported Palette Formats
+
+The palette_conv tool supports the following formats:
+
+- `.plt` - ACE's native palette format (binary, optimized for Amiga OCS)
+- `.gpl` - GIMP Palette format (text)
+- `.act` - Adobe Color Table format
+- `.pal` - ProMotion NG palette format
+- `.png` - Image preview of the palette
+
+## Command Line Usage
+
+```
+palette_conv inPath.ext [outPath.ext]
+```
+
+Where:
+- `inPath.ext` - Path to the input palette file with its extension
+- `outPath.ext` - Path to the output palette file with its extension
+
+If no output path is provided, it defaults to converting to `.gpl` format.
+
+### Example Usage
+
+```
+# Converting from GIMP palette to ACE format
+palette_conv palette.gpl palette.plt
+
+# Converting from ACE format to Adobe Color Table
+palette_conv palette.plt palette.act
+
+# Creating a PNG preview of a palette
+palette_conv palette.gpl preview.png
+```
+
+## Color Considerations
+
+ACE is designed for the Amiga OCS/ECS hardware, which uses 12-bit color (4 bits per RGB channel). When converting to ACE's native `.plt` format, the tool ensures colors are compatible with OCS limitations.
+
+When creating artwork for your game, it's recommended to:
+
+1. Use colors that work within Amiga's 12-bit color limitations (4 bits per channel)
+2. Limit your palette to the number of colors supported by your chosen bit depth
+3. Consider the Extra-Half-Bright (EHB) mode if you need more colors with limited bitplanes
```

Comment:
Having ordered list like that makes the markdown auto-increment it, lessens diffs on future changes.

The EHB point doesn't make sense to me

```suggestion
1. Use colors that work within Amiga's 12-bit color limitations (4 bits per channel)
1. Limit your palette to the number of colors supported by your chosen bit depth
```